src/puptoo/process/profile.py

```diff
@@ -171,6 +174,18 @@ def system_profile(
             catch_error("dmidecode", e)
             raise
 
+    if ansible_info:
+        try:
+            if ansible_info.catalog_worker_version:
+                profile["ansible_catalog_worker_version"] = ansible_info.catalog_worker_version
```

Comment:
the system_profile has an "ansible" object with subfields, but this has the fields at the top-level. Does this PR populate the fields correctly?